l1-contracts/contracts/upgrades/Upgrade_4844.sol

```diff
@@ -8,13 +8,15 @@ import {BaseZkSyncUpgrade, ProposedUpgrade} from "./BaseZkSyncUpgrade.sol";
 /// @author Matter Labs
 /// @custom:security-contact security@matterlabs.dev
 contract Upgrade_4844 is BaseZkSyncUpgrade {
+    address constant BLOB_VERSIONED_HASH_GETTER_ADDR = 0x0000000000000000000000000000000000001337;
```

Comment:
Wait, what?

Reply:
I think we should remove this contract because it is needed only for one time usage, and also having contract that sets `BLOB_VERSIONED_HASH_GETTER_ADDR` to random magic address doesn't make much sense to me

Reply:
what should we do with it?

Reply:
In this case just remove the file, but in case you see such dummy value it means we probably need to precaculate it somewhere and better to set it in constructor